packages/back-end/src/scim/users/listUsers.ts

```diff
@@ -51,12 +51,12 @@ export async function listUsers(
 
   return res.status(200).json({
     schemas: ["urn:ietf:params:scim:api:messages:2.0:ListResponse"],
-    totalResults: filteredUsers.length,
+    totalResults: resources.length,
     Resources: resources,
     startIndex: queryOptions.startIndex,
     itemsPerPage:
-      resources.length < queryOptions.count
-        ? resources.length
-        : queryOptions.count,
+      !resources.length || resources.length > queryOptions.count
+        ? queryOptions.count
+        : resources.length,
```

Comment:
Hmm do you need the `!resources.length || ` here?

Reply:
Looking at the documentation further, I think we can simplify this even more to just:

```ts
itemsPerPage: queryOptions.count,
```

The documentation states:

_"itemsPerPage  The number of resources returned in a list response
      page.  REQUIRED when partial results are returned due to
      pagination."_
      
So it seems like if we aren't returning partial results, the field is ignored. I'll do some testing around this today.